popup.html

```diff
@@ -10,8 +10,9 @@
 <body>
     <div class="spinner"></div>
     <div class="linkList">
-        <div>
+        <div class="buttonContainer">
             <button class="saveButton" type="button">Save Tab</button>
+            <button class="saveButton saveAllButton" type="button">Save All Tabs</button>
```

Comment:
Change this to have an id="saveAllButton" so that you do not get click event for saveButton